### DISCUSSION TITLE
Is it possible to mitm http/3 request?

### DISCUSSION THREAD

**Discussion by Karmylr** [mitmproxy/mitmproxy#5929] [Q&A]
Is it possible to mitm http/3 request?

**Comment by meitinger** [comment]
Support for _reverse_ HTTP3 exists since #5435, but is experimental.
Usage might not be as easy as with HTTP2, though, since installing mitmproxy's certificate might not be enough. (for Chromium check out `--origin-to-force-quic-on`)
Feedback is always welcome :)

**Reply by 3052** [reply]
my understanding is that reverse HTTP/3 cannot be used to intercept, so much so thats its basically not even worth mentioning. however if thats not the case I am interested.
